runtime/westend/src/lib.rs

```diff
@@ -332,16 +330,16 @@ impl pallet_election_provider_multi_phase::Config for Runtime {
 	type Currency = Balances;
 	type SignedPhase = SignedPhase;
 	type UnsignedPhase = UnsignedPhase;
-	type SolutionImprovementThreshold = MinSolutionScoreBump;
+	type SolutionImprovementThreshold = SolutionImprovementThreshold;
 	type MinerMaxIterations = MinerMaxIterations;
 	type MinerMaxWeight = OffchainSolutionWeightLimit; // For now use the one from staking.
 	type MinerTxPriority = MultiPhaseUnsignedPriority;
```

Comment:
StakingUnsignedPriority should be removed and MultiPhaseUnsignedPriority shouldn't rely on it.